autogen/docs/codetab.md.mustache

```diff
@@ -5,6 +5,8 @@ to use immediately go in the Command Center; commands you want to save and use
 later, over and over again, are found in the Code tab.
 ![[codetab.gif]]
 
+The Code tab can be opened in a separate window from the main NetLogo window. It can then be moved and resized.
+
```

Comment:
I was assuming this would go into the new code tab section, but if you want to keep it towards the top of the file, I'd move it above the code tab image with the rest of the intro text.